src/Pos/Communication/Message.hs

```diff
@@ -94,9 +94,17 @@ instance (MessagePart tag) =>
         tagM _ = Proxy
     formatMessage _ = "Request"
 
+instance (MessagePart tag) =>
```

Comment:
I think you should do it. And it's a minor change, can be done directly in `master`. The only thing for now I want you to add is this tag integer here: https://github.com/input-output-hk/cardano-docs.iohk.io/blob/master/_docs/technical/protocols/2017-01-01-application-level.md#message-names